source/gear/feverbreak-organic.js

```diff
@@ -0,0 +1,39 @@
+const { GearTemplate } = require('../classes');
+const { dealDamage, removeModifier } = require('../util/combatantUtil');
+const { listifyEN } = require('../util/textUtil');
+const { SURPASSING_VALUE } = require('../constants');
+
+module.exports = new GearTemplate("Organic Fever Break",
+    "Deals to a foe, the damage that is pending from any Poison and Frail on them, and then removes those debuffs",
```

Comment:
Don't forget to update the description to state what the Organic variant does.

Reply:
Also, the @{element} tag will interpolate to the gear's element's emoji (for marking damage element).